resemblyzer/voice_encoder.py

```diff
@@ -9,38 +9,43 @@
 
 
 class VoiceEncoder(nn.Module):
-    def __init__(self, device: Union[str, torch.device]=None, verbose=True):
+    def __init__(self, device: Union[str, torch.device]=None, verbose=True, weights_fpath: Union[Path, str]=None):
```

Comment:
Put it as first argument (and update references accordingly)

Reply:
Wouldn't that produce some back-compatibility issues if some developers will use **device** as _positional argument_, not _keyword argument_?

Reply:
Right I forgot this was Resemblyzer, backwards compat is ideal.